make/Build.java

```diff
@@ -0,0 +1,1616 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+# This program will download/build the dependencies for jtreg and then
+# build jtreg. Downloaded files are verified against known/specified
+# checksums.
+#
+# The program can be executed directly as a single source-file program
+# by the Java launcher, using JDK 12 or later.
```

Comment:
Just curious... why JDK 12?  I presume there's a goal to use the oldest release possible and there's a feature you need in 12?

Reply:
IIRC, it was rounding up from a bug fix required in a JDK 11 update.

Reply:
I'll investigate the current requirements.

Reply:
Also, we should differentiate between the version used to _build_ `jtreg` from the version required to _run_ it.